src/Storages/System/StorageSystemTables.cpp

```diff
@@ -37,8 +37,8 @@ StorageSystemTables::StorageSystemTables(const StorageID & table_id_)
         {"data_paths", std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
         {"metadata_path", std::make_shared<DataTypeString>()},
         {"metadata_modification_time", std::make_shared<DataTypeDateTime>()},
-        {"dependencies_database", std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
-        {"dependencies_table", std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
+        {"dependent_views_database", std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
```

Comment:
Will this break queries accessing the old names? If so, is it really worth it?

Reply:
I'd like to rename it, and this is a system table so it can be changed from time to time.
But it's not necessary to do that right now in this PR, so I think I'll move this change to a separate PR.

Reply:
> I'd like to rename it, and this is a system table so it can be changed from time to time.

Yes, but it's still a pain from the user point of view as it requires different queries for different CH version (and CH doesn't support SQL scripts).

If you are thinking about renaming it, why not take the opportunity to improve the API with other types? Maybe it should be an array of tuples (database, table) so you don't have to be doing `arrayZip` to use get the pair.